events-api/protobuf/v1/event.proto

```diff
@@ -101,3 +106,13 @@ enum RebuildStatus {
   // Rebuild failed
   Failed = 4;
 }
+
+// Replica event details
+message ReplicaEventDetails {
+  // Pool name
+  string pool_name = 1;
+  // Pool uuid
+  string pool_uuid = 2;
+  // lvol uuid
+  string lvol_uuid = 3;
```

Comment:
Lvol is impl specific

```suggestion
  string replica_uuid = 3;
```

Btw should it be the other way around, uuid in the target and name here?

Reply:
Yes the uuid should be in the target. 

I was looking at from<lvol> for replica in a different location. Will change it.

Reply:
Done.